include/superior_mysqlpp/dnsa_connection_pool.hpp

```diff
@@ -201,21 +239,34 @@ namespace SuperiorMySqlpp
                 jobThread = std::thread{&DnsAwarePoolManagement::job, std::cref(*this)};
             }
 
+            /**
+             * Returns true if jobThread is in joinable state.
+             */
             auto isDnsAwarePoolManagementThreadRunning() const
             {
+                // XXX: this is ambigious, it can return false if jobThread is detached
```

Comment:
Your point is corrent, however  `jobThread` is private and I can't see any detaching - so it's more like a note about possible danger? I'd reformulate it that way.

Reply:
`ambiguous`

Reply:
ACK